tests/components/application-connector/README.md

```diff
@@ -64,3 +64,12 @@ Image versions and the external service name used during testing can be set up i
    ```bash
    make test-gateway
    ```
+4. If you run the tests with debug option:
+   
+   ```bash
+   make test-gateway-debug
+   ```
+   You need to remember to turn on the sidecars as soon as the test is completed.
+   ```bash
+   make enable-sidecar-after-mtls-test
+   ```
```

Comment:
``````suggestion
   ```bash
   make test-gateway
   ```
   - Optionally, run the tests in the debugging mode:
   
      ```bash
      make test-gateway-debug
      ```
      In this mode, you must remember to enable the sidecars as soon as the test is completed:
      ```bash
      make enable-sidecar-after-mtls-test
      ```
``````

- Don't split sentences in the middle. Finish a sentence, and then provide a code block/picture/... .
- `with debug option` -> `with the debug option`: an article was missing. Is `the debugging mode` also correct? If so, let's use it.